eth/tracers/api.go

```diff
@@ -714,14 +714,10 @@ func (api *API) traceBlockParallel(ctx context.Context, block *types.Block, stat
 		})
 	}
 
-	// upgrade build-in system contract before tracing non-system tx if Feynman is not enabled
 	parent, err := api.blockByNumberAndHash(ctx, rpc.BlockNumber(block.NumberU64()-1), block.ParentHash())
 	if err != nil {
 		return nil, err
 	}
-	if !api.backend.ChainConfig().IsFeynman(block.Number(), block.Time()) {
```

Comment:
why delete this, because if not IsFeynman, we should upgrade contract code at the begining.